Yafc/Windows/WelcomeScreen.cs

```diff
@@ -165,12 +165,15 @@ protected override void BuildContents(ImGui gui) {
                     """, false)) {
                 _ = gui.BuildCheckBox("Use net production/consumption when analyzing recipes", netProduction, out netProduction);
             }
-            using (gui.EnterRowWithHelpIcon("""
-                If checked, the main project screen will not use hardware-accelerated rendering.
-                Enable this setting if YAFC crashes after loading without an error message, or if you know that your computer's graphics hardware does not support modern APIs (e.g. DirectX 12 on Windows).
-                """, false)) {
+
+            string softwareRenderHint = "If checked, the main project screen will not use hardware-accelerated rendering.\n\n" +
```

Comment:
Shouldnt this be `\r\n`?

Reply:
Oh i see we use linux style `\n` everywhere. In that case, this should probably be a single `\n`, unless the double newline was intentional?

Reply:
I tested this part in particular. The equivalent of a line break in triple-quoted string is double new-line character in a common string.

Reply:
Interesting

Reply:
> The equivalent of a line break in a triple-quoted string is double new-line character in a common string.

That's a feature in git, triggering an (IMO) bug in SDL. On Windows, git ends line with \r\n, and SDL interprets that as equivalent to \n\n.

That said, I looked at the two tooltips on the Welcome screen, and I do prefer the version with the blank line, so this will make it so the blank line also appears for Linux compilers.